_plugin-as-service/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Plugin as a Service
+nav_order: 1
+has_children: false
+has_toc: false
+nav_exclude: true
+permalink: /plugin-as-service/
+redirect_from: 
+  - /plugin-as-service/index/
+---
+
+# Plugin as a Service 🚀
+
+## Introduction
+OpenSearch allows users to extend core features using **plugins**, but traditional plugins have several **limitations**:
+- They operate in the **same JVM as a cluster**, sharing **storage, memory, and state**.
+- They require **strict version compatibility**.
+- They are **restricted to a single tenant**.
+
+To address these challenges, we introduced a **remote metadata SDK client**:
+
+> 📌 **[SDK Client Repository](https://github.com/opensearch-project/opensearch-remote-metadata-sdk)**  
+> Enables **stateless** OpenSearch plugins using **external data stores** like a remote OpenSearch cluster or cloud provider storage.
+
+This enhancement **improves scalability** and makes plugins more **adaptable** for large-scale workloads.
+
+---
+
+## 🔹 Multi-Tenancy Support
+To **increase flexibility**, OpenSearch now supports **multi-tenancy** in key plugins:
+- **[ML-Commons](https://github.com/opensearch-project/ml-commons)** (for machine learning workloads)
+- **[Flow Framework](https://github.com/opensearch-project/flow-framework/)** (for workflow automation)
```

Comment:
should we link to the opensearch.org docs for "Machine Learning" and "Automating Configurations" here?

Reply:
added.